cli/tests/pcluster/models/dummy_s3_bucket.py

```diff
@@ -87,6 +87,7 @@ def mock_bucket_object_utils(
     delete_s3_artifacts_side_effect=None,
     upload_bootstrapped_file_side_effect=None,
     check_bucket_is_bootstrapped_side_effect=None,
+    check_bucket_is_bootstrapped_return_value=None,
```

Comment:
why the check should return None, rather than True/False?

Reply:
Fixed. Changed to True. Thanks for catching!